### PR TITLE
feat: Improve makefile and readme

### PR DESCRIPTION
**Pull request by martinsaporiti**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -76,22 +84,20 @@ build-ui:
...
 # Run the api, pending_publisher and notifications services
 .PHONY: run
-run:
-	@if [ -f ./resolvers_settings.yaml ]; then \
-  		COMPOSE_DOCKER_CLI_BUILD=1 $(DOCKER_COMPOSE_CMD) up -d api pending_publisher notifications; \
-	else \
-  		echo "./resolvers_settings.yaml not found"; \
-  	fi
+run: up
+	COMPOSE_DOCKER_CLI_BUILD=1 $(DOCKER_COMPOSE_CMD) up -d api pending_publisher notifications
```
Comment (ilya-korotya) [inline Makefile:88]: Where do we check that the file `./resolvers_settings.yaml` exists?

File: README.md [hunk 2]
```diff
@@ -52,191 +51,148 @@ Streamline the **Verifiable Credentials issuance** process with the user-friendl
...
-    ```
-3. Start the infrastructure:
+```bash
+ISSUER_UI_INSECURE=true
+```
 
-    ```bash
-    make up
-    ```
+5. Import your private Key:
```
Comment (ilya-korotya) [inline README.md:102]: Is optional step? Should we do this if we keep keys locally?

**Comment by x1m3** [comment]
I'm having this error when running make run-all in a fresh install 
```
COMPOSE_DOCKER_CLI_BUILD=1 docker compose -p issuer -f /home/xime/devel/lala/issuer-node/infrastructure/local/docker-compose.yml up -d ui api pending_publisher notifications
WARN[0000] /home/xime/devel/lala/issuer-node/infrastructure/local/docker-compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion 
env file /home/xime/devel/lala/issuer-node/.env-api not found: stat /home/xime/devel/lala/issuer-node/.env-api: no such file or directory
```
Solved by running a touch .env-api before. Not sure if I'm doing something wrong, if we should document this touch in readme or if we should run this touch in some place

**Comment by martinsaporiti** [comment]
> I'm having this error when running make run-all in a fresh install
> 
> ```
> COMPOSE_DOCKER_CLI_BUILD=1 docker compose -p issuer -f /home/xime/devel/lala/issuer-node/infrastructure/local/docker-compose.yml up -d ui api pending_publisher notifications
> WARN[0000] /home/xime/devel/lala/issuer-node/infrastructure/local/docker-compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion 
> env file /home/xime/devel/lala/issuer-node/.env-api not found: stat /home/xime/devel/lala/issuer-node/.env-api: no such file or directory
> ```
> 
> Solved by running a touch .env-api before. Not sure if I'm doing something wrong, if we should document this touch in readme or if we should run this touch in some place

`.env-api` is no longer needed. Docker Compose dependency with that file has to be removed.

**Review by x1m3 — APPROVED** [review]

**Pull request merged**